python/paddle/fluid/tests/unittests/test_sigmoid_focal_loss_op.py

```diff
@@ -19,8 +19,7 @@
 import numpy as np
 from op_test import OpTest
 
-import paddle.fluid as fluid
-from paddle.fluid import Program, core, program_guard
+from paddle.fluid import core
```

Comment:
There is no need to use `fluid.core`. In this file, all `core.xxx` can be replaced with `paddle.xxx`